client/components/mma/holiday/HolidayReview.tsx

```diff
@@ -138,15 +141,17 @@ export const HolidayReview = () => {
 								}}
 							>
 								<Checkbox
-									checked={isCheckboxConfirmed}
-									onChange={(newValue) =>
-										setIsCheckboxConfirmed(newValue)
-									}
 									label={
 										productType.holidayStops
 											.explicitConfirmationRequired
 											.checkboxLabel
 									}
+									checked={isCheckboxConfirmed}
+									onChange={() => {
+										setIsCheckboxConfirmed(
```

Comment:
Is this a case where we should be [passing a function](https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state) to `setIsCheckboxConfirmed`, since we're updating the state based on the current state? In the old pre-hooks world I think I would have done this, but not sure if it's required here. e.g. `setIsCheckBodyConfirmed(previousIsCheckboxConfirmed => !!previousIsCheckboxConfirmed)`

Reply:
hey, I think we're ok here as we are only calling the setState method once and not multiple times in a row/tick.  Good shout though